eng/packaging.targets

```diff
@@ -38,4 +38,11 @@
     <MSBuild Projects="$(PkgProjPath)"
              Targets="Build" />
   </Target>
+
+  <ItemGroup>
+    <DefaultValidateFramework Include="$(NetCoreAppCurrent)">
```

Comment:
We should make this change in arcade.  OK to do so here for now, but please open an issue to track making it in arcade.

Reply:
Yeah thats the plan :P

Reply:
Ok, please file issue and add a comment